Frontail optional fails to install

Not sure if this is platform specific (using Buster on RPi 4)

```
2019-08-06_17:23:01_BST [openHABian] Installing the openHAB Log Viewer (frontail)...
$ npm install -g frontail
npm ERR! path /usr/lib/node_modules/frontail
npm ERR! code EISGIT
npm ERR! git /usr/lib/node_modules/frontail: Appears to be a git repo or submodule.
npm ERR! git     /usr/lib/node_modules/frontail
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-08-06T16_23_10_187Z-debug.log
FAILED (frontail)
```

This can be fixed by removing the `/usr/lib/node_modules/.git directory` and trying again, but I'm not sure if this is an issue for openHABian or an upstream repo somewhere.

--- a/functions/nodejs-apps.bash
+++ b/functions/nodejs-apps.bash
@@ -30,16 +30,21 @@ nodejs_setup() {
 frontail_setup() {
   nodejs_setup
   echo -n "$(timestamp) [openHABian] Installing the openHAB Log Viewer (frontail)... "
+  frontail_base="/usr/lib/node_modules/frontail"
+  # clear frontail install dir if it exists, otherwise npm install will fail, #647
+  if [ -d "$frontail_base" ]; then 
+    # rm -rf "$frontail_base"
+    cond_redirect npm uninstall -g frontail
+  fi
   cond_redirect npm install -g frontail
   if [ $? -ne 0 ]; then echo "FAILED (frontail)"; exit 1; fi
   cond_redirect npm update -g frontail
   #
-  frontail_base="/usr/lib/node_modules/frontail"
   mkdir -p ${frontail_base}/preset ${frontail_base}/web/assets/styles
   
-  cp $BASEDIR/includes/frontail-preset.json $frontail_base/preset/openhab.json
-  cp $BASEDIR/includes/frontail-theme.css $frontail_base/web/assets/styles/openhab.css
-  cp $BASEDIR/includes/frontail.service /etc/systemd/system/frontail.service
+  cp "$BASEDIR"/includes/frontail-preset.json $frontail_base/preset/openhab.json
+  cp "$BASEDIR"/includes/frontail-theme.css $frontail_base/web/assets/styles/openhab.css
+  cp "$BASEDIR"/includes/frontail.service /etc/systemd/system/frontail.service
   chmod 664 /etc/systemd/system/frontail.service
   cond_redirect systemctl daemon-reload
   cond_redirect systemctl enable frontail.service

